README.md

```diff
@@ -1,39 +1,40 @@
+## Retrieval Augmented Generation (RAG)
 
-By leveraging an AI workflow for your specific use case, you can streamline development of AI solutions following the example provided by the workflow to:
+A RAG pipeline embeds multimodal data --  such as documents, images, and video -- into a database connected to a Large Language Model.  RAG lets users to chat with their own data. 
 
-- Reduce development time, at lower cost
-- Improve accuracy and performance
-- Gain confidence in outcome, by leveraging NVIDIA AI expertise
+| Name                                                                                                                 | LLM   | Framework  | Multi-GPU | Multi-Node | Embedding | TRT-LLM | Triton                                                                                                    | VectorDB  | K8s                                                                                                                                          |
+|------------------------------------------------------------------------------------------------------------------------|-------------|------|-----------|------------|----------|------|-----------------------------------------------------------------------------------------------------------|------|---------------------------------------------------------------------------------------------------------------------------------------------|
+| [Developer RAG](https://github.com/NVIDIA/GenerativeAIExamples/tree/main/RetrievalAugmentedGeneration)                       | llama2-13b     | Langchain  | Yes       | No        | e5large-v2| Yes    | Yes    | Milvus  | No                                                                                                                                           |
+| [Developer RAG](https://github.com/NVIDIA/GenerativeAIExamples/tree/main/RetrievalAugmentedGeneration)                       | llama2-13b     | Llama Index  | Yes       | No        | e5large-v2| Yes    | Yes    | Milvus  | No                                                                                                                                           |
+ 
 
-Using the example workflow provided in this repository, you know exactly what AI framework to use, how to bring data into the pipeline, and what to do with the data output. AI Workflows are designed as microservices, which means they can be deployed on Kubernetes alone or with other microservices to create a production-ready application for seamless scaling. The workflow cloud deployable package can be used across different cloud instances and is automatable and interoperable.
+## Large Language Models
+The NVIDIA family of Large Language Models (LLMs) is optimized for building production-ready generative AI applications for the enterprise.
```

Comment:
Llama2 is OSS LLM.

Reply:
I am going to add links to the Nemotron models on HF in this table.